src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java

```diff
@@ -12,6 +12,7 @@
     private final Address address;
     private final boolean addressAmended;
     private final Address correspondenceAddress;
+    private final String mobilePhone;
```

Comment:
isn't this just supposed to be phoneNumber now? we don't ask for a mobile, just a contact number

Reply:
Good point, we should rename it at some point but doing it now would trigger too many changes, both in frontend and backend.